lib/folio_client.rb

```diff
@@ -51,7 +51,7 @@ def get(path, params = {})
   # @param path [String] the path to the Folio API request
   # @param request [json] request body to post to the API
   def post(path, request = nil)
-    response = connection.post(path, request, { "x-okapi-token": config.token })
```

Comment:
I ran `standardrb --fix` since this gem uses standardrb and it made some linting fixes, not related to this issue.

Reply:
this is good, but 
- it's almost always better to put the linting fixes in a separate PR
- puttting them in a separate commit means they can be separated from other review

Reply:
Noted, thanks